browser/ui/views/side_panel/mobile_view/mobile_view_side_panel_manager.cc

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/side_panel/mobile_view/mobile_view_side_panel_manager.h"
+
+#include <utility>
+
+#include "brave/browser/ui/views/side_panel/mobile_view/mobile_view_side_panel_coordinator.h"
+#include "chrome/browser/ui/browser_window/public/browser_window_interface.h"
+#include "ui/views/view.h"
+
+MobileViewSidePanelManager::MobileViewSidePanelManager(
+    BrowserWindowInterface* browser_window_interface)
```

Comment:
shouldn't this be `BrowserWindowInterface& browser_window_interface`? because you're passing into raw_ref.

Reply:
Yup, changed.